.github/workflows/close_stale_issues.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'There has not been any recent activity on this ticket, so we are marking it as stale. Please comment if you have any additional questions.'
+          days-before-issue-stale: 14
+          days-before-pr-stale: -1
+          days-before-close: 1
```

Comment:
can we make the window here a little longer so people have more of a window to follow up, like 1 week?

Reply:
I updated it to 7 days for stale and 7 for closing for a total of 14 days to match our jira policy.